source/usage-examples/insertOne.txt

```diff
@@ -0,0 +1,89 @@
+.. _csharp-insert-one:
+
+=================
+Insert a Document
+=================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can insert a single document into a collection by using the synchronous
+``InsertOne()`` method, or the asynchronous ``InsertOneAsync()`` method.
+
+Examples
+--------
+
+These examples use the following ``Restaurant`` class as a model:
+
+.. literalinclude:: ../includes/code-examples/insert-one/Restaurant.cs
+   :start-after: start-model
+   :end-before: end-model
+   :language: csharp
+   :copyable:
+   :dedent:
+
+The following example inserts a document into 
+the ``restaurants`` collection.
+
+Select the :guilabel:`Asynchronous` or :guilabel:`Synchronous` tab to see the 
+corresponding code.
+
+.. tabs::
+
+   .. tab:: Asynchronous
+      :tabid: insert-one-async
+
+      .. literalinclude:: ../includes/code-examples/insert-one/InsertOneAsync.cs
+         :start-after: start-insert-one-async
+         :end-before: end-insert-one-async
+         :language:  csharp
+         :copyable:
+         :dedent:
+
+      For a fully runnable example of the ``InsertOneAsync()`` operation, see the
+      `Asynchronous Insert One Example <{+example+}/InsertOneAsync.cs>`__.
+
+   .. tab:: Synchronous
+      :tabid: insert-one-sync
+
+      .. literalinclude:: ../includes/code-examples/insert-one/InsertOne.cs
+         :start-after: start-insert-one
+         :end-before: end-insert-one
+         :language:  csharp
+         :copyable:
+         :dedent:
+
+      For a fully runnable example of the ``InsertOne()`` operation, see the
+      `Synchronous Insert One Example <{+example+}/InsertOne.cs>`__.
+
+Expected Result
+~~~~~~~~~~~~~~~
+
+After running either of the preceding full examples, the ``InsertOne()``
+method inserts the document, and the :ref:`Find() <csharp-find-one>` method returns
+the newly inserted document. The output is as follows:
+
+.. code-block:: none
+
+   // results truncated
```

Comment:
I read this:
![image](https://user-images.githubusercontent.com/44870738/210857673-6d32a3f7-a545-4c1a-9d95-1e7b0afde7a9.png)
like the commented text was added in output which is not true, can we formulate it better?